lib/auth/trustedcluster.go

```diff
@@ -439,7 +439,16 @@ func (a *Server) GetRemoteClusters(opts ...services.MarshalOption) ([]services.R
 	return remoteClusters, nil
 }
 
-func (a *Server) validateTrustedCluster(validateRequest *ValidateTrustedClusterRequest) (*ValidateTrustedClusterResponse, error) {
+func (a *Server) validateTrustedCluster(validateRequest *ValidateTrustedClusterRequest) (resp *ValidateTrustedClusterResponse, err error) {
+	defer func() {
+		if err != nil {
+			log.Infof("Trusted cluster validation failed: %v", err)
```

Comment:
log.WithError(err).Warningf("Failed to validate trusted cluster token")

Reply:
done